src/components/hero/Hero.js

```diff
@@ -1,92 +1,13 @@
 import React from "react";
 
 import ButtonComponent from "../Button/ButtonComponent";
-
-import { Grid, Typography, makeStyles } from "@material-ui/core";
+import useStyles from "./style";
+import { Grid, Typography } from "@material-ui/core";
 import LanguageIcon from "@material-ui/icons/Language";
 import MoreVertIcon from "@material-ui/icons/MoreVert";
 import ArrowBackIcon from "@material-ui/icons/ArrowBack";
 import ReactPlayer from "react-player/lazy";
 
-const useStyles = makeStyles((theme) => ({
```

Comment:
@Devesh21700Kumar Not creating an extra css file was also under one of the reason to use makeStyles. When our app will expand it will be harder to manage css and js seperately in two files for all components.

Reply:
i guess an extra jss file would actually be beneficial.
because there were projects with 1000 lines of code in each js file and beleive me,. styling in a different page for such expandable components was something that really brought relief.

Anyways if u want then i can surely put them back

thanks for the review

Reply:
So should i put them back?

Reply:
Yes,
We will be breaking pages into smaller components, and those smaller components have everything with them like html, css and js,

I guess you should revert that part.

Reply:
Surei.. I'll do it in a few minutes

Reply:
@Abhishek-kumar09 have done it.. pls see and merge